features/bootstrap.feature

```diff
@@ -263,3 +263,69 @@ Feature: Bootstrap WP-CLI
       """
       foo
       """
+
+  @require-wp-3.9
+  Scenario: Run cache flush on ms_site_not_found
+    Given a WP multisite install
+    And a wp-cli.yml file:
+      """
+      url: invalid.com
+      """
+
+    When I try `wp cache add foo bar`
+    Then STDERR should contain:
+      """
+      Error: Site 'invalid.com' not found.
+      """
+    And the return code should be 1
+
+    When I run `wp cache flush --url=invalid.com`
+    Then STDOUT should contain:
+      """
+      Success:
+      """
+    And the return code should be 0
+
+  @require-wp-3.9
+  Scenario: Run search-replace on ms_site_not_found
+    Given a WP multisite install
+    And a wp-cli.yml file:
+      """
+      url: invalid.com
+      """
+
+    When I try `wp search-replace foo bar`
+    Then STDERR should contain:
+      """
+      Error: Site 'invalid.com' not found.
+      """
+    And the return code should be 1
+
+    When I run `wp search-replace foo bar --network`
```

Comment:
Noticed that if you change this to something that will get replaced, eg `wp search-replace example bar --network` then `Search_Command` will throw a PHP fatal as it relies on the WP functions `esc_sql()` (and `like_escape()`) being available so will need to shim these in `Search_Command`.

Reply:
@gitlost But `Search_Command` runs after WordPress has loaded, which means WordPress would be loaded for `wp search-replace`?

Reply:
I haven't followed the logic but it's probably some multisite peculiarity in loading - if you make that change to the test do you see the fatal?

Reply:
Oh, I see what you're referring to:

![image](https://user-images.githubusercontent.com/36432/33155285-2d14e5da-cfa4-11e7-95b1-880508cddf39.png)

Reply:
They're trivial functions so I'll do a PR on `search-command` with them copied in...

Reply:
I think we can / should accommodate in Runner. Doing so now.

Reply:
Ok - will hold fire...

Reply:
7be7c4d2df2bd6d31656800eff99ba80e3d13eb3

Reply:
Cool. (Shimming those functions might still be something to consider as it leaves `search-replace` only dependent on `$wpdb`, which could be useful.)